src/main/ruby/core/exception.rb

```diff
@@ -166,6 +166,19 @@ def to_s
 end
 
 class UncaughtThrowError < ArgumentError
+  attr_reader :tag
+  attr_reader :value
+
+  def initialize(*args)
+    Rubinius::Type.check_arity(args.size, 2, -1)
```

Comment:
I don't like manual arity checks as you know :)
Can it be expressed by with the method arguments/signature?

Reply:
@eregon Fixed, I've updated the method signature.